src/highrl/envs/teacher_env.py

```diff
@@ -86,6 +87,10 @@ def _get_robot_metrics(self) -> None:
         """Calculates and prints training session results indicating how well the robot performed
         during this session. This is done for every robot trainig session created by the teacher.
         """
+        len_results_str = f"length of results: {len(self.opt.results)}"
+        results_str = f"results: {self.opt.results}"
+        _LOG.debug(len_results_str)
+        _LOG.debug(results_str)
```

Comment:
It is better to have print statements starting with capital letters:
- `"length of results: {len(self.opt.results)}"` -> `"Length of results: {len(self.opt.results)}"`
- `"results: {self.opt.results}"` -> `"Results: {self.opt.results}"`